samples/enable-autoupgrade-changes/readme.txt

```diff
@@ -0,0 +1,5 @@
+This sample illustrates changes required to provide autoupgrade of Minor version as a choice to the end user.

```

Comment:
Can you please add more details in the intro?

Automatic Extension Upgrade is available for Marketplace Kubernetes applications (extensions) for minor and patch version upgrades. When 'autoUpgradeMinorVersion' setting is enabled, the application is upgraded automatically whenever the application publisher releases a new minor or patch version for that application. This sample illustrates changes required to provide auto upgrade of minor version as a choice to the end user at deployment time. 

At the end of the README, can we add below in a NOTE section?
Please note this sample includes only a small subset of the files. The files in this sample folder contains the additional changes required on top of the base sample application 'k8s-offer-azure-vote'.

Reply:
@thomasyip-msft , @maanasagovi , @atchutbarli  - can you please review above text especially the end section?
For every example, in the README, please add a introduction and at the footer note, refer the base application.